creation/host-devices.md

```diff
@@ -0,0 +1,59 @@
+Host Devices Assignment
+=======================
+
+KubeVirt provides a mechanism for assigning host devices to a virtual machine.
+This mechanism is generic and allows various types of PCI devices, such as GPU
+or any other devices attached to a PCI bus,  to be assigned. It also allows
+Mediated devices, such as pre-configured virtual GPUs to be assigned using the
+same mechanism.
+  
```

Comment:
Maybe a small note that today the slicing/pre-configured virtual GPUs needs to be done outside of kubevirt?

Reply:
Done